pygmt/src/wiggle.py

```diff
@@ -0,0 +1,92 @@
+"""
+wiggle - Plot z=f(x,y) anomalies along tracks.
+"""
+from pygmt.clib import Session
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    J="projection",
+    R="region",
+    Z="scale",
+    B="frame",
+    D="position",
+    G="color",
+    T="track",
+    U="timestamp",
+    V="verbose",
+    W="pen",
+    X="xshift",
+    Y="yshift",
```

Comment:
Sort alphabetically.

```suggestion
    B="frame",
    D="position",
    G="color",
    J="projection",
    R="region",
    T="track",
    U="timestamp",
    V="verbose",
    W="pen",
    X="xshift",
    Y="yshift",
    Z="scale",
```